### ISSUE TITLE
todo - 05.22

### ISSUE THREAD

**Issue by sieu-n** [krenerd/awesome-modular-pytorch-lightning#2]
Goals:

- [ ] Implement Faster R-CNN and object detection.
  - [ ] Dataloader
  - [ ] Training RPN
  - [ ] Combined loss and fine-tuning training.
  - [ ] make predictions and mAP metric.
  - [ ] Utilities: visualization

- [ ] Implement continual learning
  - [ ] load weights from pre-trained model.
  - [ ] linear evaluation task with weights fixed
